subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxMetaMapBuilder.groovy

```diff
@@ -172,7 +172,7 @@ class JmxMetaMapBuilder {
     private static ObjectName getObjectName(def map) {
         if (!map) return null
         def jmxName
-        if (map.name instanceof String) {
+        if (map.name instanceof String || map.name instanceof GString) {
```

Comment:
Would "instanceof CharSequence" cover it?

Reply:
I'll merge but tweak with the suggestion from @eric-milles

Reply:
Thanks for the suggestion.